concrete-core/src/backends/core/private/crypto/secret/generators/encryption.rs

```diff
@@ -314,3 +314,51 @@ fn noise_bytes_per_ggsw(
 ) -> usize {
     level.0 * noise_bytes_per_ggsw_level(glwe_size, poly_size)
 }
+
+#[cfg(all(test, feature = "multithread"))]
+mod test {
+    use crate::backends::core::private::crypto::bootstrap::StandardBootstrapKey;
+    use crate::backends::core::private::crypto::secret::generators::{
+        EncryptionRandomGenerator, SecretRandomGenerator,
+    };
+    use crate::backends::core::private::crypto::secret::{GlweSecretKey, LweSecretKey};
+    use concrete_commons::dispersion::Variance;
+    use concrete_commons::parameters::{
+        DecompositionBaseLog, DecompositionLevelCount, GlweSize, LweDimension, PolynomialSize,
+    };
+
+    #[test]
+    fn test_gaussian_sampling_margin_factor() {
```

Comment:
I don't see any `assert` in this test, If I understand its because this is to test that the function does not panic, maybe it could be made explicit in the name `test_gaussian_sampling_margin_factor_does_not_panic` 🤔

Reply:
@blowfish880 I agree with @tmontaigu on this one :wink:

Reply:
I also agree.

Reply:
Yep good one thanks ! I changed the name for your proposition @tmontaigu .